starter-code/contact-app/src/app/contact-list/contact-list.component.ts

```diff
@@ -11,14 +11,23 @@ export class ContactListComponent implements OnInit {
   newContact: Object = {};
 
   constructor() { }
-
+  
   ngOnInit() {
     this.contacts = contactList;
   }
 
+  onSubmit(event){
+    event.preventDefault();
+  }
+
   addContact(){
     console.log("Add contact has been called");
+
+     this.contacts.push(this.newContact);
```

Comment:
Aquí hubiera sido interesante que solo hicieran push al arreglo solo si hay algo en `this.newContact` 
así se evita tener contactos vacíos.